microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -149,9 +149,12 @@ def _install(self) -> bool:
             self.node.log.info(
                 "Installing dpdk and dev package from package manager..."
             )
-            if isinstance(node.os, Ubuntu) or isinstance(node.os, Redhat):
+            if isinstance(node.os, Ubuntu):
                 node.os.install_packages(["dpdk", "dpdk-dev"])
                 self._dpdk_version_info = node.os.get_package_information("dpdk")
+            elif isinstance(node.os, Redhat):
+                node.os.install_packages(["dpdk", "dpdk-devel"])
+                self._dpdk_version_info = node.os.get_package_information("dpdk")
```

Comment:
this line is sharable.

Reply:
I thought that too but it has to be in the isinstance clause bc of the type checker.

Reply:
How about assert it's posix before the if clause?

Reply:
Oo good point, I can use self.can_install to assert